src/tests/EdFi.DataManagementService.Tests.E2E/Features/General/URLValidation.feature.disabled

```diff
@@ -0,0 +1,385 @@
+                      "gradeLevels": [
+                          {
+                              "gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#Transitional Kindergarten"
+                          }
+                      ],
+                      "schoolId": 2244668800,
+                      "nameOfInstitution": "Institution Test"
+                  }
+                  """
+             Then it should respond with 404
+              And the response body is
+                  """
+                  {
+                      "detail": "The specified data could not be found.",
+                      "type": "urn:ed-fi:api:not-found",
+                      "title": "Not Found",
+                      "status": 404,
+                      "correlationId": null
+                  }
+                  """
+              And the headers contain
+                  """
+                  Content-Type: application/problem+json
+                  """
+
+        @ignore
+        Scenario: 07 - Ensure clients cannot update a resource when endpoint does not end in plural
+            Given the system has the following "schools"
+                  | schoolId  | nameOfInstitution        | gradeLevelDescriptor                             | educationOrganizationCategoryDescriptor                        |
+                  | 255901044 | Grand Bend Middle School | uri://ed-fi.org/GradeLevelDescriptor#Sixth grade | uri://ed-fi.org/EducationOrganizationCategoryDescriptor#School |
+             When a PUT request is made to "/ed-fi/school/{id}" with
+                  """
+                  {
+                      "educationOrganizationCategories": [
+                          {
+                              "educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor#School"
+                          }
+                      ],
+                      "gradeLevels": [
+                          {
+                              "gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#Transitional Kindergarten"
+                          }
+                      ],
+                      "schoolId": 2244668800,
+                      "nameOfInstitution": "Institution Test"
+                  }
+                  """
+             Then it should respond with 404
+              And the response body is
+                  """
+                  {
+                      "detail": "The specified data could not be found.",
+                      "type": "urn:ed-fi:api:not-found",
+                      "title": "Not Found",
+                      "status": 404,
+                      "correlationId": null
+                  }
+                  """
+              And the headers contain
+                  """
+                  Content-Type: application/problem+json
+                  """
+
+        @ignore
+        Scenario: 08 - Ensure clients cannot delete a resource when endpoint does not end in plural
```

Comment:
I think these plural tests are not required in this way, because if the url is not valid it will return the 404... If I'm not wrong we already have tests when the url is invalid. What we can do is to find specific urls where the 's' at the end is not required for irregular plurals. 

@stephenfuqua could you confirm if these can be removed?

Reply:
That is a good point, but I think there is something valuable about keeping these. The domain entity is called `School`, and these tests are confirming that the _endpoints_ only use the plural.

Reply:
Keeping the scenario